README.md

```diff
@@ -493,6 +506,29 @@ responder.on('convert', (req, cb) => {
 </p>
 </details>
 
+</p>
+</details>
+
+<details>
+<summary>
+    Click to see the complete <code>client.js</code> file.
```

Comment:
this example originally takes places much earlier in the readme, around line 239, where we actually didn't need to add the concept of a `key`. I'm not sure if it's a good idea to update it here. How should we tackle this?

Is changing the example to remove the necessity to introduce keys here be a solution?

Reply:
I will create an example of arbitration service and push it to new repo, so we can than copy it to example folder. Maybe that is the better approach. I have some time now to do this

Reply:
https://github.com/matejthetree/cote-conversion-service/blob/master/README.md